docs/src/main/sphinx/release/release-374.md

```diff
@@ -0,0 +1,16 @@
+# Release 374 (16 Mar 2022)
+
+## General
+
+* Improve performance of queries with `GROUP BY` clauses. ({issue}`11392`)
+* Improved handling for queries where individual tasks require lots of memory.
```

Comment:
@losipiuk @martint  .. could we cut this shorter to only be 

Improve retry handling for queries where individual tasks require lots of memory. 


The rest feels like documentation and @losipiuk and @jhlodin are working on this now to get ready soon. Ideally a first version even with the 374 release..

Reply:
+1, this behavior is covered in https://github.com/trinodb/trino/pull/11404

Could reword to "Expanded :doc:`/admin/fault-tolerant-execution` to improve handling for queries where individual tasks require lots of memory" as a way to sneak a link in there.

Reply:
I would rephrase along the lines of:

> Improve handling of queries where individual tasks require lots of memory when `retry-policy` is set to `TASK`